File: src/main/java/kr/co/vilez/appointment/model/dto/MyAppointListDto.java
package kr.co.vilez.appointment.model.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class MyAppointListDto {
    private int id;
    private int userId;
    private String title;
    private String hopeAreaLat;
    private String hopeAreaLng;
    private String startDay;
    private String endDay;
    private String appointmentId;
    private String shareUserId;
    private String notShareUserId;
    private int type;
    private String date;
    private String appointmentStart;
    private String appointmentEnd;
}

File: app/src/main/java/kr/co/vilez/data/dto/MyAppointListDTO.kt
package kr.co.vilez.data.dto

data class MyAppointListDTO(
    val appointmentEnd: String,
    val appointmentId: String,
    val appointmentStart: String,
    val date: String,
    val endDay: String,
    val hopeAreaLat: String,
    val hopeAreaLng: String,
    val id: Int,
    val notShareUserId: String,
    val shareUserId: String,
    val startDay: String,
    val title: String,
    val type: Int,
    val userId: Int
)